ss/test/storage_test_suite.go

```diff
@@ -141,7 +141,7 @@ func (s *StorageTestSuite) TestDatabaseGetVersionedKey() {
 	// all queries after version 15 should return nil
 	for i := int64(15); i <= 17; i++ {
 		bz, err = db.Get(storeKey1, i, key)
-		s.Require().NoError(err)
+		s.Require().Error(err)
```

Comment:
also assert expected wrapped error type? 

Ditto in other places

Reply:
Updated on latest commit